source/help/apps/desktop-guide.rst

```diff
@@ -0,0 +1,176 @@
+===================================
+Desktop Guide
+===================================
+
+Mattermost desktop applications are available for Windows, Mac and Linux operation systems. 
+
+You can `download the apps directly from our download page <https://about.mattermost.com/downloads/>`_ and you can visit our `installation guides <https://docs.mattermost.com/install/desktop.html>`_ for help during set up.
+
+Below are a few tips to get you started and configuring your experience on the desktop app:
+
+ - `Team Management <https://docs.mattermost.com/help/apps/desktop-guide.html#team-management>`_
+ - `Settings <https://docs.mattermost.com/help/apps/desktop-guide.html#settings>`_
+ - `Menu Bar <https://docs.mattermost.com/help/apps/desktop-guide.html#menu-bar>`_
+
+Team management
+---------------------------------------------------------------------
+
+You can connect to multiple Mattermost servers from a single interface on the desktop app.
+
+**Adding servers and teams**
+
+To add a new server on the Mattermost instance, 
+
+1. Click **Add new team** next to the right of Team Management section.
+2. Enter **Name** and a valid **URL**, which begins with either ``http://`` or ``https://``.
+3. Click **Add**.
+
+You can now access all teams you have joined in the server once you have saved the settings, and each will appear as a separate tab in the app. 
+
+You may also add a separate instance for each team, if you'd like.
+
+**Editing servers and teams**
+
+To edit a server, 
+
+1. To the right of the team or server you want update, click **Edit**.
+2. Edit **Name** and/or **URL**.
+3. Click **Save**.
+
+**Removing servers and teams**
+
+To remove a server from your desktop app environment, click on **Remove** next to the server or a team you want removed. Note that this action cannot be undone.
+
+Settings
+---------------------------------------------------------------------
+
+In addition to `Mattermost Account Settings <https://docs.mattermost.com/help/settings/account-settings.html>`_ , the desktop app provides additional options to customize your experience. 
+
+The Settings Page is available from the `File` menu under `Settings`, and can be accessed with `CTRL + COMMA` on Windows and Linux, and with `CMD + COMMA` on Mac.
+
+Description of each setting is included below this table.
+
++-------------------------------------------------------------------------------+---------------------------+---------------------------+---------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------+
```

Comment:
Seems weird to have contributors mentioned in documentation, as it's potentially confusing to end users why it's there. 

Propose a separate CREDITS page to thank people involved. I think as the project grows larger, it might just be a list of names, rather than individual call outs--maybe even link to it from the product. 

Propose seeing what other projects use, e.g. [Mozilla](https://www.mozilla.org/credits/), 

![image](https://cloud.githubusercontent.com/assets/177788/18618904/0a0b40c2-7da5-11e6-8233-17faea0ccc8e.png)